Supervision Work/src/uk/ac/cam/ap886/oopjava/supervisionProjects/Hangman/CLI.java

```diff
@@ -0,0 +1,209 @@
+package uk.ac.cam.ap886.oopjava.supervisionProjects.Hangman;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class CLI implements UserInterface {
+	
+	
+	Game game;
+	BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+	
+	
+	//----------------
+	
+	public CLI (Game newGame){
+		game = newGame;
+	}
+	
+	
+	
+	private void prn(String arg){
```

Comment:
An alternative would have been `import System.out` and then you could just use `println`